catalog/catalog.yaml

```diff
@@ -290,6 +287,21 @@ components:
         media:
           $ref: '#/components/schemas/Media'
 
+    CatalogDescriptor:
```

Comment:
How's `GeneratedCatalog` or `CatalogWrapper` as the name? CatalogDescriptor sounds like it's a type that lives alongside or within the Catalog schema, not as a superset of it.

Reply:
I think I like `CatalogWrapper` slightly better, since `Generated` is situational (not always the case). I'll use that.